code/modules/unit_tests/strange_reagent.dm

```diff
@@ -0,0 +1,151 @@
+/// Checks to ensure strange reagent works as expected
+/datum/unit_test/strange_reagent
+	priority = TEST_LONGER
+
+	var/datum/reagent/medicine/strange_reagent/instant/strange_reagent
+	var/target_max_health
+	var/amount_needed_to_full_heal
+	var/amount_needed_to_revive
+
+/datum/unit_test/strange_reagent/Run()
+	strange_reagent = new
+
+	var/list/types_to_check = typecacheof(list(
+		/mob/living/carbon/human,
+		/mob/living/simple_animal,
+		/mob/living/basic,
+	))
+	types_to_check -= /mob/living/simple_animal/pet/gondola/gondolapod // need a pod, which we don't have
+	types_to_check -= typesof(/mob/living/simple_animal/hostile/megafauna) // no
+	types_to_check -= typesof(/mob/living/basic/mouse) // qdel themselves on death; why dont they use DEL_ON_DEATH you might ask. I-unno
+	types_to_check -= typesof(/mob/living/simple_animal/slime) // if they roll the 50% chance to spawn as an adult, they can just at random split and qdel themselves
+
+	for(var/mob/living/type as anything in types_to_check)
+		var/mob/living/target = allocate_new_target(type)
+		var/is_basic = istype(target, /mob/living/basic)
+		var/is_simple = istype(target, /mob/living/simple_animal)
+		// check some basic stuff
+		if(target.status_flags & GODMODE)
+			continue
+		if(!(target.mob_biotypes & MOB_ORGANIC))
+			continue
+
+		if(is_simple)
+			var/mob/living/simple_animal/simple_animal = target
+			if(simple_animal.del_on_death)
+				continue
+			simple_animal.loot?.Cut()
+
+		if(is_basic)
+			var/mob/living/basic/basic = target
+			if(basic.basic_mob_flags & DEL_ON_DEATH)
+				continue
+
+		if(istype(target, /mob/living/carbon))
+			var/mob/living/carbon/carbon = target
+			for(var/obj/item/bodypart/limb as anything in carbon.bodyparts)
+				limb.body_damage_coeff = 1 // fuck you
```

Comment:
It's not obvious to me at all what this is for

Reply:
I think this was me just getting mad at limbs for having different body damage coeffs from when I kept getting errors; gonna comment this out to see if it causes failures still